documentation/modules/oauth/proc-oauth-kafka-config.adoc

```diff
@@ -95,10 +95,12 @@ spec:
     disableTlsHostnameVerification: true <1>
     checkAccessTokenType: false <2>
     accessTokenIsJwt: false <3>
+    scope: any <4>
 ----
 <1> (Optional) Disable TLS hostname verification. Default is `false`.
 <2> If the authorization server does not return a `typ` (type) claim inside the JWT token, you can apply `checkAccessTokenType: false` to skip the token type check. Default is `true`.
 <3> If you are using opaque tokens, you can apply `accessTokenIsJwt: false` so that access tokens are not treated as JWT tokens.
+<4> If the authorization server requires the client to specify the `scope` when requesting the token from the token endpoint, you can provide it - in this case it is 'any'.
```

Comment:
Can we use `any` with any authorization server that requires a scope? Or is `any` the default for authorizationservers that don't need a specific value?

Reply:
No, this is just an example. There are no standard specified possible valiues. It can be anything - up to how security is configured on the server.